news/4003.dev

```diff
@@ -0,0 +1 @@
+Make Test Cases look clickable.
```

Comment:
Ah, sorry, I didn't notice in my previous review the news fragment type: this isn't a development related change, a filename suffix of `.bug` should be fine.